stdlib/cmake/modules/AddSwiftStdlib.cmake

```diff
@@ -1346,6 +1346,18 @@ function(add_swift_target_library_single target name)
       target_link_options(${target} PRIVATE
         "LINKER:-current_version,${SWIFT_COMPILER_VERSION}")
     endif()
+
+    # In the past, we relied on unsetting globally
+    # CMAKE_OSX_ARCHITECTURES to ensure that CMake
+    # would# not add the -arch flag. This is no longer
```

Comment:
```suggestion
    # would not add the -arch flag. This is no longer
```

Reply:
Thanks for noticing the typo -- this should be addressed in d03731bf6d4